blog/2015/10/graduate_communities.html

```diff
@@ -0,0 +1,63 @@
+---
+layout: blog
+root: ../../..
+author: Damien Irving
+title: "Inserting Software Carpentry graduates into coding communities"
+date: 2015-10-19
+time: "08:00:00"
+category: ["Opinion"]
+---
+<!-- start excerpt -->
+<p>
+  I was chatting to Greg Wilson via email the other day,
+  when I got sidetracked on a long-winded rant about the big issues facing Software Carpentry.
+  (I might be one of the reasons Greg has trouble clearing his inbox...) 
+  I thought I'd share an excerpt from my email here, 
+  as a possible solution to the issue of supporting learners 
+  after they've attended one of our workshops:
+</p>
+<!-- end excerpt -->
+<p>
+  "Besides the instructor training bottleneck, 
+  it seems to me that the other huge issue facing Software Carpentry
+  is what happens to learners after they've attended a workshop. 
```

Comment:
Use this as the opening paragraph, and cut the lead to, "One of the issues I'd like Software Carpentry to tackle is what happens to learners <em>after</em> they've attended a workshop."